socorro/external/es/crashstorage.py

```diff
@@ -211,6 +211,77 @@ def save_raw_and_processed(self, raw_crash, dumps, processed_crash,
 
 
 #==============================================================================
+class ESCrashStorageRestrictedDump(ESCrashStorage):
+    """This class stores redacted crash reports into Elasticsearch, but instead
+    of removing the entire `json_dump`, it keeps only a subset of its keys.
+    """
+    required_config = Namespace()
+    required_config.add_option(
+        name="json_dump_keys",
+        doc="keys of the json_dump field to keep in the processed crash",
+        default=[
+            "largest_free_vm_block",
+            "tiny_block_size",
+            "write_combine_size",
+        ],
```

Comment:
One thing I've never understood about configman, and how we use configman in socorro, is when do you write the list as a string and when you actually write a python list. For example, the `forbidden_keys` thing is a string that gets converted to a python list by configman.